tests/phpunit/tests/media.php

```diff
@@ -3929,6 +3995,9 @@ static function( $attr ) {
 	 *
```

Comment:
```suggestion
	 * @ticket 58235
	 *
	 * @since 6.3.0 Reset counting variables and applied `fetchpriority="high"` to `$header_img`.
```
Added the ticket number that made this change and moved noted the change in a `@since` annotation.